person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name: 'Unknown', parent_permission: 'Unknown')
+    @id = Ramdom.rand(1..1000)
```

Comment:
- [ ] Kindly note that the instance variable `@parent_permission` should be a boolean and should be true by default, not `unknown`.

Reply:
This issue is well resolved! ✔️